### DISCUSSION TITLE
Enable unlimited amount of numbers in Maska?

### DISCUSSION THREAD

**Discussion by brandonleichty** [beholdr/maska#111] [Q&A] [answered]
Hello! 

Is there a config / pattern for allowing allowing _only_ numbers to be typed? For example, I could use `data-maska="#####"` to allow up to five numbers to be typed. But what if I don't want to hard code the value - and allow the user to type numbers until the `maxlength` of the input is reached? 

Thank you!

**Comment by beholdr** [comment, marked as answer]
Yes, you can use custom token with `multiple` flag for that

``` html
<input v-maska data-maska="0" data-maska-tokens="0:[0-9]:multiple">
```

**Reply by brandonleichty** [reply]
Thanks a ton for the quick reply!
